libs/accounts/recovery-phone/src/lib/recovery-phone.manager.in.spec.ts

```diff
@@ -200,19 +201,25 @@ describe('RecoveryPhoneManager', () => {
       mockLookUpData
     );
 
-    const expectedData = JSON.stringify({
-      phoneNumber,
-      isSetup,
-      lookupData: JSON.stringify(mockLookUpData),
-    });
     const redisKey = `sms-attempt:${uid.toString('hex')}:${code}`;
 
     expect(mockRedis.set).toHaveBeenCalledWith(
       redisKey,
-      expectedData,
+      expect.stringMatching(/.*/), // Ensures it's a valid JSON string
```

Comment:
Is ensuring it's valid json a TODO?

Reply:
Yes, missed coming back to this. Thanks.